tests/components/generic_hygrostat/test_humidifier.py

```diff
@@ -687,7 +687,7 @@ async def test_init_ignores_tolerance(hass, setup_comp_3):
     calls = await _setup_switch(hass, True)
     _setup_sensor(hass, 39)
     await hass.async_block_till_done()
-    assert 1 == len(calls)
+    assert len(calls) == 1
     call = calls[0]
     assert HASS_DOMAIN == call.domain
     assert SERVICE_TURN_OFF == call.service
```

Comment:
Does it not work on constants?
```suggestion
    assert call.domain == HASS_DOMAIN
    assert call.service == SERVICE_TURN_OFF
```

Reply:
Interesting, a bug maybe? Will take a look

Reply:
I haven't looked into it, maybe it's by design.
Maybe it doesn't work with constants, or maybe it only works with constants marked as "Final".

Reply:
Fixed in https://github.com/charliermarsh/ruff/pull/2255

Reply:
If you wish, we can merge this one and tackle the new ones when we update `ruff`.

Reply:
Ruff moves really fast, let's see what happens upstream.

Reply:
It was merged, also added https://github.com/charliermarsh/ruff/pull/2266

To handle attribute calls to constants on imported modules or things like enums.